source/includes/fact-request-private-preview.rst

```diff
@@ -0,0 +1,4 @@
+1. In the :guilabel:`Code Generation` tab, click the 
```

Comment:
[question]

Should we use the `.. procedure::` directive here?

Reply:
I did try and stage this up with the procedure directive, but the step names were bolding the content unexpectedly and given the short nature of the steps I think we should leave this as is, this should be removed in the near future once the feature is published.

<img width="799" alt="Screenshot 2023-10-05 at 12 39 50 PM" src="https://github.com/mongodb/docs-relational-migrator/assets/85948430/f52274b7-b57e-45a1-aea8-30252392a9b8">

Reply:
Got it. I think this is ok to leave as-is. For future reference, I find the `.. procedure` styling that's not connected works better for short procedures.

As in using:

```
.. procedure::
   :style: normal
```

instead of

```
.. procedure::
   :style: connected
```

Reply:
Appreciate you pointing this out, I never noticed this subtle difference in the directive.